gen/definitions/bgp_address_family.yaml

```diff
@@ -46,24 +46,144 @@ attributes:
+    default_value: "noncrit"
+    example: 8000
+  - nxos_name: advertL2vpnEvpn
+    tf_name: adv_l2vpn_evpn
+    type: String
+    description: "Enable or disable the advertisement of L2VPN EVPN routes."
+    enum_values:
+      - enabled
+      - disabled
+    default_value: disabled
+    example: disabled
+  - nxos_name: advPip
+    tf_name: adv_phyip_for_type5_routes
+    type: String
+    description: "Advertise physical IP for type-5 routes"
+    enum_values:
+      - enabled
+      - disabled
+    default_value: disabled
+    example: disabled
+  - nxos_name: maxEcmp
+    tf_name: max_ecmp_paths
+    type: Int64
+    description: "Maximum number of ECMP paths."
+    min_int: 1
+    max_int: 128
+    default_value: 1
+    example: 2
+  - nxos_name: maxExtEcmp
+    tf_name: max_ext_ecmp_paths
+    type: Int64
+    description: "Maximum number of external ECMP paths."
+    min_int: 1
+    max_int: 128
+    default_value: 1
+    example: 1
+  - nxos_name: maxExtIntEcmp
+    tf_name: max_ext_int_ecmp_paths
+    type: Int64
+    description: "Maximum number of external/internal ECMP paths."
+    min_int: 1
+    max_int: 128
+    default_value: 1
+    example: 1
+  - nxos_name: maxLclEcmp
+    tf_name: max_eqcost_ecmp_paths
+    type: Int64
+    description: "Maximum number of equal-cost multipath for local paths ECMP paths."
+    min_int: 1
+    max_int: 128
+    default_value: 1
+    example: 1
+  - nxos_name: maxMxdEcmp
+    tf_name: max_mixcost_ecmp_paths
     type: Int64
-    description: 'The next-hop address tracking delay timer for non-critical next-hop reachability routes.'
+    description: "Maximum mixed equal-cost multipath for local and remote ECMP paths."
     min_int: 1
-    max_int: 4294967295
-    default_value: 10000
-    example: 1800
+    max_int: 128
+    default_value: 1
+    example: 1
+  - nxos_name: defInfOriginate
+    tf_name: def_inf_originate
+    type: String
+    description: "Enable or disable the default-information originate."
+    enum_values:
+      - enabled
+      - disabled
+    default_value: disabled
+    example: disabled
+  - nxos_name: nhRtMap
+    tf_name: next_hop_route_map_name
+    type: String
+    description: "Next hope route map name"
+    example: ROUTEMAP1
+    default_value: ""
+  - nxos_name: prfxPriority
+    tf_name: prefix_priority
+    type: String
+    description: "Enable prefix priority for AF"
+    enum_values:
+      - none
+      - high
+    default_value: none
+    example: none
+  - nxos_name: retainRttAll
+    tf_name: retain_rt_all
+    type: String
+    description: "Retain Route Target All"
+    enum_values:
+      - enabled
+      - disabled
+    default_value: disabled
+    example: disabled
+  - nxos_name: supprInactive
+    tf_name: adv_only_act_routes
+    type: String
+    description: "Advertise only active routes to peers"
+    enum_values:
+      - enabled
+      - disabled
+    default_value: disabled
+    example: disabled
+  - nxos_name: tblMap
```

Comment:
I think the feature on the cli is name "table-map", so maybe `table_map_route_map_name`.